### DISCUSSION TITLE
TMP1200

### DISCUSSION THREAD

**Discussion by madwheeling** [artisan-roaster-scope/artisan#585] [Q&A] [answered]
Does artisan work with more than 2 TMP1200 sensors?

**Comment by MAKOMO** [comment, marked as answer]
> On 11. Mar 2021, at 00:36, madwheeling <notifications@github.com> wrote:
> 
> 
> Does artisan work with more than 2 TMP1200 sensors?

Yep, but you can have only two different configurations (eg. PT100 in one group and PT1000 in the second group).

M

**Reply by madwheeling** [reply]
That is great news. How do I set up different groups? I am only seeing TMP1200 A and B in the Phidgets section and the extra devices section.

**Reply by CarefreeBuzzBuzz** [reply]
@madwheeling you answered your own question.  Group A and Group B.  Two RTDs of the same type in each group.

**Comment by MAKOMO** [comment]
> On 18. Mar 2021, at 17:43, madwheeling ***@***.***> wrote:
> 
> 
> That is great news. How do I set up different groups? I am only seeing TMP1200 A and B in the Phidgets section and the extra devices section.

Those are the two groups. You can set the configuration (like the probe type) of each of those in the Phidgets tab (menu Config >> Device, 4th tab). Then per device channel you select either the TMP1200 A or the TMP1200B to use the one or the other configuration.

M

**Reply by madwheeling** [reply]
Thank you for your help, I think I understand now. I just need to find a PT1000 probe that will work for my 3rd sensor now.
